tests/PHPStan/Rules/Classes/ClassConstantRuleTest.php

```diff
@@ -19,6 +19,8 @@ class ClassConstantRuleTest extends RuleTestCase
 
 	private int $phpVersion;
 
+	private bool $checkNonStringableDynamicAccess;
```

Comment:
If all the test cases use `true` then this property isn't needed at all. Just use `true` in the `new` expression in `getRule`.